tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
+
+This tutorial will explain how to get notified by [Slack](https://slack.com/) of various events that originate within [Google Cloud Platform](https://cloud.google.com/). `Google Compute Engine` instances may start and stop, new `Google AppEngine` verions might be deployed and `Google Cloud Storage` bucket may be created or deleted. People would like to be notified of these and others via their `Slack` account.
+
+`gSlack` allows you to install a lightweight serverless solution into your
+own Google Cloud Platform project/s and easily configure the alerts you would like to receive.
+
+## Objectives
+In this tutorial you will learn how to:
+- Enable `Slack` integration.
+- Create `Google PubSub` a topic.
+- Create `Google Stackdriver` log exports that write into the `PubSub` topic.
+- Configure permissions for the log exports to be written into the `PubSub`.
+- Configure and deploy a `Google CloudFunctions`(**beta**) serverless backend code to call the `Slack` API.
+- Configure the alerts to notify of.
+
+## Before you begin
+This tutorial runs best when using a `*nix` operating system.
+
+If you don't have access to a `*nix` machine you may choose to use `Google Cloud Shell` using this [quickstart](https://cloud.google.com/shell/docs/quickstart).
+### Prerequisites
+- You must have a `Google Cloud Platform` project with enabled billing.
+- You must have access to a user with `Owner` role on the project.
+- You must have `gcloud` SDK and CLI installed on your machine ([quickstart](https://cloud.google.com/sdk/docs/quickstarts)).
+- You must have a `Slack` account.
+
+**Please notice that some of the services used in this tutorial are still `beta versions` at the time of this writing and are subject to `different or no SLA`. Please consult the documentation before using this tutorial with any production critical workflow.**
+
+### TL;DR
+If you wish to skip this step-by-step tutorial - you can use the quick setup by following the [README.md](./README.md)
+
+## Setting up your project
+
+**Please remember to put your project ID where the text indicates so with: '`<YOUR_PROJECT_ID>`'**
```

Comment:
Suggested edit:
\*\*Please remember to put your project ID where the text indicates `to do so`: '\`<YOUR_PROJECT_ID>\`'\*\*